### ISSUE TITLE
Drop usage of the word "extension"

### ISSUE THREAD

**Issue by jmeas** [jmeas/resourceful-redux#63]
I should stop calling things "extensions," which makes them sound like some formal Concept©. To reduce the number of Concepts©, I should just update that section of the docs to be called "Ecosystem," then refer to them as libraries that can be used with Resourceful Redux.

**Comment by jmeas** [comment]
Still thinking on this. I may keep it, but maybe not. Feel free to chime in if you feel one way or the other. For now, I'm going to close this

**Comment by smaifullerton-wk** [comment]
@jmeas Darn, I was working on this. :P 

In many cases you refer to Extensions at the same time as the core library:

`Not in the core library, but there is an [Action Creators extension]` 

This makes the changing Extensions to `library` a bit confusing: 

`Not in the core library, but there is an [Action Creators library]` 

I was thinking something like: 

`Not in the core library, but there is a collection of [Action Creators]`

Then you can simplify a lot of sentences like

`The Action Creators extension provides out-of-the-box action creators that you can use to create resources.`

to this:

`Resourceful Redux provides [a collection of] out-of-the-box Action Creators that you can use to create resources.`

And the README change:

```
# Ecosystem

The Resourceful Redux ecosystem provides bits of code that make working with 
the library even better.

* [Prop Types](/docs/extensions/prop-types.md)
* [Action Creators](/docs/extensions/action-creators.md)
```

**Comment by jmeas** [comment]
yo @smaifullerton-wk ! Thanks for the comment!

> @jmeas Darn, I was working on this. :P

I didn't mean to make it out that I'm not interested in making the change! If you think it's a good idea, then let's do it!!! ✌️ 

> I was thinking something like:
> 
> `Not in the core library, but there is a collection of [Action Creators]`

I agree with you that using the word "library" could lead to confusion. I'm not opposed to using another noun instead of "library," and I think "a collection of action creators" is 👌 . Another idea that might be worth considering is giving the extensions a proper name, derived from the npm package name.

The npm package name for the extensions is `resourceful-{thing}` (like `resourceful-action-creators`, so I was thinking the name of them could be, say, "Resourceful Action Creators."

Then, in some places it could be referred to directly by that name:

> Not in the core library, but there is a separate module, Resourceful Action Creators.

```
# Ecosystem

The Resourceful Redux ecosystem provides bits of code that make working with 
the library even better.

* [Resourceful Prop Types](/docs/extensions/resourceful-prop-types.md)
* [Resourceful Action Creators](/docs/extensions/resourceful-action-creators.md)
```

Maybe it makes sense to sometimes refer to it by name, and other times with the more general "collection of action creators," based on the context? I'm not too sure – just throwin' some ideas out there.

If you made a PR that got rid of "extensions" (in whatever way you think is best), I'd def merge it though :v:

Thanks again!

**Comment by jmeas** [comment]
Resolved in #128
